pkgs/servers/tailscale/default.nix

```diff
@@ -36,7 +36,6 @@ buildGoModule rec {
   meta = with lib; {
     homepage = "https://tailscale.com";
     description = "The node agent for Tailscale, a mesh VPN built on WireGuard";
-    platforms = platforms.linux;
```

Comment:
Why was this removed? Shouldn't this be `platforms.linux ++ platforms.darwin`?

Reply:
See: https://github.com/NixOS/nixpkgs/pull/140058#discussion_r719596196

Reply:
Oh right :facepalm: